x-pack/plugins/ml/server/routes/results_service.ts

```diff
@@ -265,4 +283,65 @@ export function resultsServiceRoutes({ router, mlLicense }: RouteInitialization)
       }
     })
   );
+
+  /**
+   * @apiGroup ResultsService
+   *
+   * @api {get} /api/ml/results/:jobId/categorizer_stats
+   * @apiName GetCategorizerStats
+   * @apiDescription Returns the categorizer snapshots for the specified job ID
+   * @apiSchema (params) getCatergorizerStatsSchema
+   */
+  router.get(
+    {
+      path: '/api/ml/results/{jobId}/categorizer_stats',
+      validate: {
+        params: jobIdSchema,
+        query: getCategorizerStatsSchema,
+      },
+      options: {
+        tags: ['access:ml:canGetJobs'],
+      },
+    },
+    mlLicense.fullLicenseAPIGuard(async (context, request, response) => {
+      try {
+        const resp = await getCategorizerStats(context, request.params, request.query);
+        return response.ok({
+          body: resp,
+        });
+      } catch (e) {
+        return response.customError(wrapError(e));
+      }
+    })
+  );
+
+  /**
+   * @apiGroup ResultsService
+   *
+   * @api {get} /api/ml/results/:jobId/categorizer_stats
```

Comment:
should be 
```suggestion
   * @api {post} /api/ml/results/stopped_partitions
```

Reply:
Updated here https://github.com/elastic/kibana/pull/74592/commits/a94ecaf246b7f84168eb296b5c2c1a75471353cc